src/components/__tests__/BannerInfo.cy.js

```diff
@@ -0,0 +1,70 @@
+import BannerInfo from 'components/global/BannerInfo.vue';
+
+describe('<BannerInfo>', () => {
+  context('desktop', () => {
+    beforeEach(() => {
+      cy.fixture('bannerInfo').then((bannerInfo) => {
+        cy.wrap(bannerInfo).as('bannerInfo');
+        cy.mount(BannerInfo, {
+          props: {
+            title: bannerInfo.title,
+            icon: bannerInfo.icon,
+          },
+          slots: {
+            default: bannerInfo.content,
+          },
+        });
+        cy.viewport('macbook-16');
+      });
+    });
+
+    coreTests();
+  });
+
+  context('mobile', () => {
+    beforeEach(() => {
+      cy.fixture('bannerInfo').then((bannerInfo) => {
+        cy.wrap(bannerInfo).as('bannerInfo');
+        cy.mount(BannerInfo, {
+          props: {
+            title: bannerInfo.title,
+            icon: bannerInfo.icon,
+          },
+          slots: {
+            default: bannerInfo.content,
+          },
+        });
+        cy.viewport('iphone-6');
+      });
+    });
+
+    coreTests();
+  });
+});
+
+function coreTests() {
+  it('renders component', () => {
+    cy.get('@bannerInfo').then((bannerInfo) => {
+      // component
+      cy.dataCy('banner-info').should('be.visible');
+      // icon
+      cy.dataCy('banner-info-icon')
+        .should('be.visible')
+        .and('have.css', 'font-size', '96px');
```

Comment:
Does it really test the image loading (which is important)? Same as in previous PR #439, #438 and maybe in previous closed PRs.

Example https://stackoverflow.com/questions/51246606/test-loading-of-image-in-cypress#58641462.

```js
// we can wait for the <img> element to appear
// but the image has not been loaded yet.
cy.get('[alt="delayed image"]')
.should('be.visible')
.and(($img) => {
  // "naturalWidth" and "naturalHeight" are set when the image loads
  expect($img[0].naturalWidth).to.be.greaterThan(0)
})

```

Reply:
Try disable load additional icons themes inside component.js file and run test again. Test doesn't catch state when icon is not loaded.